flocker/provision/_ssh/_conch.py

```diff
@@ -154,7 +153,8 @@ def perform_run_remotely(reactor, base_dispatcher, intent):
 
     def connect():
         connection = connection_helper.secureConnection()
-        connection.addErrback(lambda _: False)
+        connection.addErrback(write_failure)
```

Comment:
Yes, and I see a couple places in the existing code that rely on the same assumption.